graphql/connection_test.go

```diff
@@ -1276,6 +1274,55 @@ func TestPaginatedFilters(t *testing.T) {
 				return false
 			}),
 	)
+
+
+	inner.FieldFunc("innerConnectionWithFilterWithArgs", func(args Args) []Item {
+		return []Item{
+			{Id: 1, FilterText: "can", String: "a"},
+			{Id: 2, FilterText: "man", String: "a"},
+			{Id: 3, FilterText: "cannot", String: "a"},
+			{Id: 4, FilterText: "soban", String: "a"},
+			{Id: 5, FilterText: "socan", String: "a"},
+			{Id: 6, FilterText: "aan", String: "a"},
+			{Id: 7, FilterText: "jan", String: "a"},
+			{Id: 8, FilterText: "ban", String: "a"},
+			{Id: 9, FilterText: "dan", String: "a"},
+			{Id: 10, FilterText: "ean", String: "a"},
+			{Id: 11, FilterText: "fan", String: "a"},
+			{Id: 12, FilterText: "gan", String: "a"},
+		}
+
+	}, schemabuilder.Paginated,
+		schemabuilder.BatchFilterField("filterTextBatched",
+			func(items map[batch.Index]Item, args Args) (map[batch.Index]string, error) {
```

Comment:
Do you mind providing a test case in which the user args are used for filtering as well as one for sorting? I think it would serve as useful documentation. Personally, I am also a little unclear on the use case for providing user arguments to filter and sort functions